### PR TITLE
chore: add missing test for use-scrollable-shades

### PR DESCRIPTION
**Pull request by the-teacher**
[SPT-2585]

### Description

Add missing test for use-scrollable-shades effect

### How to test

`yarn test:setup jest packages/picasso/src/ModalContent/hooks/test.tsx`

### Screenshots
Screen Shot 2022-04-12 at 23 13 13

### Review

- [x] Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/contribution/component-api.md)
-  Annotate all `props` in component with documentation
-  Create `examples` for component
-  Ensure that deployed demo has expected results and good examples
- [x] Ensure that tests pass by running `yarn test`
-  Ensure that visuals tests pass by running `yarn test:visual`. If not - check the documentation [how to fix visual tests](https://github.com/toptal/picasso/blob/master/docs/contribution/visual-testing.md#fixing-broken-visual-tests-inside-a-pr)
- Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

[SPT-2585]: https://toptal-core.atlassian.net/browse/SPT-2585?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by sanex3339 — APPROVED** [review]
👍

File: packages/picasso/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
...
+### Minor Changes
+
+
+- [#2633](https://github.com/toptal/picasso/pull/2633) [`0b7bc1b`](https://github.com/toptal/picasso/commit/0b7bc1b) Thanks [@the-teacher](https://github.com/the-teacher)! - ---
+
+  ### Modal
+
+  - `useScrollableShades` hook was covered with tests.
+  - With the commit [`c617c7d4`](https://github.com/toptal/picasso/commit/c617c7d4) the hook `useScrollableShades` was fixed to hide bottom shade for zoomed views.
+
```
Comment (TomasSlama) [inline packages/picasso/CHANGELOG.md]: why do we need to update the changelog manually? 🤔

File: packages/picasso/src/ModalContent/hooks/test.tsx [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+  const arrangeTest = (ref: testRefProps) => {
+    const {
+      result: { current }
+    } = renderHook(() =>
+      useScrollableShades(ref as unknown as RefObject<HTMLDivElement>)
+    )
+
+    return current
+  }
```
Comment (TomasSlama) [inline packages/picasso/src/ModalContent/hooks/test.tsx:62]: 👍 💯

File: packages/picasso/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,17 @@
...
```
Comment (TomasSlama) [inline packages/picasso/CHANGELOG.md:14]: why do we need to update the changelog manually? 🤔

Comment (OndrejTuma) [reply packages/picasso/CHANGELOG.md:14]: Please use `yarn changeset` instead 🙏

Comment (TomasSlama) [reply packages/picasso/CHANGELOG.md:14]: I think for adding new tests we don't even need any bump of Picasso, right?

File: packages/picasso/src/ModalContent/hooks/test.tsx [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+
+  describe('when no current object', () => {
+    it('returns top = false and bottom = false', () => {
+      const testRef = getTestRef(undefined)
+      const { top, bottom } = arrangeTest(testRef)
```
Comment (denieler) [inline packages/picasso/src/ModalContent/hooks/test.tsx:67]:
```suggestion
      const { top, bottom } = renderScrollableShadesHook(testRef)
```
maybe like this?

File: packages/picasso/CHANGELOG.md (hunk 3 above)
Comment (the-teacher) [reply packages/picasso/CHANGELOG.md:14]: Looks like I didn't get how it works. Wanted to add an information

**Review by TomasSlama — APPROVED** [review]

**Review by OndrejTuma — APPROVED** [review]
👍

File: .changeset/dry-eyes-pump.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
+---
+'@toptal/picasso': patch
+---
+
+### Modal
```
Comment (denieler) [inline .changeset/dry-eyes-pump.md:5]:
```suggestion
---
### Modal
```

Comment (denieler) [reply .changeset/dry-eyes-pump.md:5]: according to changesets guidelines

**Review by denieler — APPROVED** [review]

File: .changeset/dry-eyes-pump.md (hunk 5 above)
Comment (denieler) [reply .changeset/dry-eyes-pump.md:4]: nope, not enough 🤔 https://github.com/toptal/picasso/blob/master/docs/contribution/changeset-guidelines.md#summary-of-the-change-changelog-message

**Review by ozgurkececioglu — APPROVED** [review]

**Pull request merged**